introduction.md

```diff
@@ -1,6 +1,9 @@
 Dojo Rules
 Ruby 
 MotherFucker
+asdasdaasdasdijuas
+asfj
+sasdaasdasd
 ==========
 * Always be on time
 * Always take off your shoes
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?